articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+## About UKCloud for Microsoft Azure
+
+UKCloud for Microsoft Azure harnesses the innovation of Microsoft’s Azure platform and we deliver this service from within our award-winning government-grade multi-cloud environment – addressing the service, connectivity and diversity requirements of UK public sector organisations and their industry partners. The service is based on the Microsoft Azure Stack product, which has been developed by Microsoft as an extension of the Azure public cloud.
+
+## Service options
+
+You can build a tailored solution to meet your requirements by mixing and matching service options. The choice does not lock you to one service or size, allowing you to scale resources up or down as your requirements change.
+
+By default customers will initially have access to:
+
+- XX vCPU
+- YY GB memory
+- ZZ GB storage
+
+The options below are all currently available within the service:
+
+## IaaS Compute 
+Virtual machines (VMs) provide the basic building block providing compute resource within UKCloud for Microsoft Azure. Each standard machine has a pre-allocated number of vCPU's, amount of memory and a temporary disk. The available sizes and families of VM's are contiually increasing, broadly, the following machine families are supported:
+
+- Basic and Standard A-series. Have CPU performance and memory configurations best suited for entry level workloads like development and test.  
+- Standard D and DS Series. Have optimised CPU-to-memory configuration making them suitable for most general-purpose applications. A subset of the D-series VMs also sport higher memory per CPU making them suitable for applications that require higher amounts of memory. 
+- Basic and Standard F series. Are optimised for CPU intensive workloads so have a higher CPU to memory configurations.
+
+The full list of supported machines can be found [here](https://docs.microsoft.com/en-gb/azure/azure-stack/user/azure-stack-vm-sizes).
+
+Customers can deploy VMs using the UKCloud for Microsoft Azure portal and other tooling.
+
+> [!Warning]
+> Each VM contains a temporary disk. The temporary disk provides short-term storage for applications and processes and is intended to only store data such as page or swap files. Data on the temporary disk may be lost during a maintenance event or when you redeploy a VM. 
+
+## Storage Options 
+There are several ways in which storage can be addressed in UKCloud for Microsoft Azure:
+
+- Azure blob storage (Page)
+Page blobs are optimized for representing IaaS disks and supporting random writes that is up to 1 TB in size. An Azure Stack virtual machine attached IaaS disk is a Virtual Hard Disk (VHD) stored as a page blob. At present only unmanaged disks can be offered in the service.
+
+- Azure blob storage (Block)
+Block blobs (generically known as object storage) is optimized for streaming and storing cloud objects, and are a good choice for storing documents, media files, backups, and etc.
```

Comment:
Block blobs ... are optimised (are not is, s not z in optimised)

change and etc to and so on